pkg/cmd/target.go

```diff
@@ -345,6 +366,22 @@ func resolveNameGarden(reader ConfigReader, name string) (matches []string) {
 	return matches
 }
 
+// resolveGardenNameFromURL resolve garden name from provided dashboard URL
+func resolveGardenNameFromURL(reader ConfigReader, dashboardURL string) string {
+	config := reader.ReadConfig(pathGardenConfig)
+	gardenName := ""
+	for _, garden := range config.GardenClusters {
+		if strings.Contains(garden.DashboardURL, dashboardURL) {
+			gardenName = garden.Name
+		}
+	}
+	if gardenName == "" {
+		fmt.Println("Provided URL could not match a garden")
```

Comment:
```suggestion
		fmt.Println("a garden could not be matched for the provided dashboard url")
```
to be in line with one of the other error messages (`a garden could not be matched for the provided server address`)

Reply:
thanks, i've fixed this.